guides/common/modules/proc_migrating_project_or_smartproxy_using_backup.adoc

```diff
@@ -0,0 +1,34 @@
+= Migrating {Project} or {SmartProxy} to {EL}{nbsp}9
+
+You can migrate your existing {Project} and {SmartProxy} installation from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+The migration process involves creating a backup of the existing {Project} and {SmartProxy} installation, which is then restored or cloned on the new {EL}{nbsp}9 system.
+Note that cloning only works for {Project} backups and not for {SmartProxy} backups.
+endif::[]
+ifndef::satellite[]
+The migration process involves creating a backup of the existing {Project} and {SmartProxy} installation, which is then restored on the new {EL}{nbsp}9 system.
+endif::[]
+In the below procedure, the "Source server" refers to the existing {EL}{nbsp}8 system on which {Project} or {SmartProxy} is installed, while "Target server" refers to the new {EL}{nbsp}9 system to which {Project} or {SmartProxy} will be migrated.
+
+.Procedure:
+. Ensure that you have enough space for the backup.
+For more information, see {AdministeringDocURL}administering_red_hat_satellite/backing-up-satellite-server-and-capsule_admin#Estimating_the_Size_of_a_Backup_admin[Estimating the size of a backup] in _{AdministeringDocTitle}_.
+. Perform a full backup of the {Project} or {SmartProxy}.
+For more information, see {AdministeringDocURL}backing-up-satellite-server-and-capsule_admin#Performing_a_Full_Backup_admin[Performing a full backup of Satellite Server or Capsule Server] in _{AdministeringDocTitle}_.
+. Deploy a new {EL}{nbsp}9 system.
+Choose the appropriate method to install {Project} or {SmartProxy}:
+** To install {ProjectServer} from a connected network, follow the procedures in {InstallingServerDocURL}[{InstallingServerDocTitle}].
+ifdef::satellite[]
+** To install {ProjectServer} from a disconnected network, follow the procedures in {InstallingServerDisconnectedDocURL}[{InstallingServerDisconnectedDocTitle}].
+endif::[]
+** To install a {SmartProxyServer}, follow the procedures in {InstallingSmartProxyDocURL}[{InstallingSmartProxyDocTitle}].
+ifdef::satellite[]
+. Use either of these two procedures to restore backup from {Project}: 
+** {AdministeringDocURL}restoring-satellite-server-or-capsule-from-a-backup_admin[Restoring Satellite Server or Capsule Server from a backup] in _{AdministeringDocTitle}_.
+** {AdministeringDocURL}cloning_satellite_server#sec-Cloning_to_Target[Cloning to the target server] in _{AdministeringDocTitle}_.
+. To restore the {SmartProxy} backup, follow the steps in {AdministeringDocURL}restoring-satellite-server-or-capsule-from-a-backup_admin[Restoring Satellite Server or Capsule Server from a backup] in _{AdministeringDocTitle}_. 
+endif::[]
+ifndef::satellite[]
+. Restore the backups from {Project} or {SmartProxy}. For more information, see {AdministeringDocURL}restoring-satellite-server-or-capsule-from-a-backup_admin[Restoring Satellite Server or Capsule Server from a backup] in _{AdministeringDocTitle}_.
```

Comment:
The links here and elsewhere include hard-coded "satellite". I think that's causing issues for non-Satellite builds. Can you please check?